security/tests/integration/kubernetes_utils.go

```diff
@@ -0,0 +1,371 @@
+// Copyright 2017 Istio Authors
```

Comment:
It would be good to move this util library into tests/util

Reply:
Do you think it makes sense, or maybe we can do it later :)

Reply:
pilot has a K8s util frameworks for the integration. I want to evaluate the library first.
we can decide it later.